src/Command/Diff.php

```diff
@@ -86,25 +86,21 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         $handler = new DiffHandler($envName, $cwd, $output->isDecorated());
+        $errorIo = $this->getErrIo($input, $output);
```

Comment:
What's the motivation behind that? Since stderr is just a shortcut for "standard error output" (even if it's quite misleading) I don't think that shorten `error` to `err` would make it better